dev/VSIX/Extension/Cs/Dev17/Standalone/source.extension.vsixmanifest

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
A diff on the first line indicates a change to line endings or the BOM. Check your editor settings.